templates/scripts/create-db-user.sh

```diff
@@ -6,41 +6,46 @@ usage () {
   exit 1
 }
 
+[[ "$1" == "devenv" ]] && DEVENV=$1
```

Comment:
This is too obscure. Let's just accept SECRET_NAME and use that, and rely on the caller to set it.